wiki/entry_kettenbach.qmd

```diff
@@ -0,0 +1,30 @@
+
+"® Relationship:"
+To put this article in context with other findings in the research field, an article by Meier and Reinecke (2021) will be briefly looked at. 
+Meier et al. (2020) used the extended two-continua model of mental health created by Meier and Reinecke (2021) to conduct their literature review. Building on previous models, the idea is to integrate both perspectives; the psychopathology (PTH)  and psychological wellbeing (PWB). Thus, the model considers the positive and negative sides of mental health in relation to CMC. PTH refers to a dysfunction in behaviour of a person’s psychological functioning whereas PWB states how well a person is doing and their psychological function are working optimal. The authors claim that PWB doesn’t mean the absence of PTH and the other way round.  However, different individual and environmental influences affect PTH and PWB, which is why they also have a variety of unique indicators that fluctuate. Furthermore, Meier and Reinecke (2021) integrate “main dimensions and manifestations of both PTH and PWB, as well as risk and resilience factors, in an Extended Two-Continua Model of Mental Health’ (p. 1190). This model allows mental health indicators to be identified across a wide range of mental health research.
+Both articles use different approaches to structure the analysis; Meier et al. (2021) define multiple levels at which CMC can be analyzed. Firstly, the devices that Users use to communicate plays a role. The second level analyses the types of application that SNS provide such as emails or messages and another level analyses brands separately. Fourth level analyses CMC channels individually by their features such as Facebook’s status update or Instagram’s photos. Lastly the interactions and individual messages get looked at. The other article separates the analysis into the following nine core topics; Internet addiction and problematic Internet use, Facebook and SNS, Mobile and Smartphone use, Relationships and CMC, Chatting and texting, Cyberbullying, ICT adoption, Work-related SMS and lastly ICT use and sleep. Due to these different focuses in research results may vary or emphasize different aspects. 
+
+
+"(Q) Questions:" 
+1.	Which are further connections between CMC and MH that should receive more attention in future research? 
+2.	What role does autonomy play in the impact that CMC can have on MH?
```

Comment:
Please try to answer this question based on the paper that you have selected. This is the question that we are interested in ;-)

Reply:
The questions above include issues and topics that are not addressed in this particular literature. Though they were talked about in the limitations of the the present study and need to be further looked at in future research.
1. The paper emphasises that future research should try looking at both negative and positive effects CMC can have on MH. Therefore the authors highlight the need to change the focus of further research and thus to look at the effects of CMC in connection with MH and not only one specific aspect. 
2. The selected paper does not specifically talk about the role autonomy plays in the impact that CMC can have on MH. The authors selected nine core topics which they individually analysed in connection with the effect those can have on mental health. It is noted that future research should do a more in-depth and systematic synthesis of research from some of the broader topics this study looked at.

Reply:
Research area of your choice:
**Digital Media Use**

APA7 citation of the selected paper:
**Meier, A., Domahidi, E., & Günther, E. (2020). Computer-Mediated Communication and Mental Health. In The Oxford Handbook of Digital Technology and Society (pp. 78–110). Oxford University Press.**

Reply:
What exactly does the MH model say in the context of this study?